### PR TITLE
add linux rpm/deb announcement

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

File: content/blog/eclipse-temurin-linux-installers-available/index.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    ```
+1. Install the Temurin version you require:
+    ```bash
+    apt-get update # update if you haven't already
+    apt-get install temurin-17-jdk
+    ```
+
+## Centos/Rhel/Fedora Instructions
+
+1. Add the RPM repo to `/etc/yum.repos.d/adoptium.repo` making sure to change the Centos version if you are not using Centos 8. RPM’s are also available for Rhel and Fedora. To check the full list of versions supported take a look at https://packages.adoptium.net/ui/repos/tree/General/rpm.
```
Comment (smlambert) [inline content/blog/eclipse-temurin-linux-installers-available/index.md:54]:
```suggestion
## Centos/RHEL/Fedora Instructions

1. Add the RPM repo to `/etc/yum.repos.d/adoptium.repo` making sure to change the Centos version if you are not using Centos 8. RPM’s are also available for RHEL and Fedora. To check the full list of versions supported take a look at https://packages.adoptium.net/ui/repos/tree/General/rpm.
```

File: content/blog/eclipse-temurin-linux-installers-available/index.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    ```
+1. Install the Temurin version you require:
+    ```bash
+    yum update # update if you haven't already
+    yum install temurin-17-jdk
+    ```
+
+## openSUSE/Sles Instructions
+
+1. Import the RPM repo making sure to change the openSUSE version if you are not using OpenSUSE 15.2. RPM’s are also available for Sles 12 and 15. To check the full list of versions supported take a look at https://packages.adoptium.net/ui/repos/tree/General/rpm.
```
Comment (smlambert) [inline content/blog/eclipse-temurin-linux-installers-available/index.md:73]:
```suggestion
## openSUSE/SLES Instructions

1. Import the RPM repo making sure to change the openSUSE version if you are not using OpenSUSE 15.2. RPM’s are also available for SLES 12 and 15. To check the full list of versions supported take a look at https://packages.adoptium.net/ui/repos/tree/General/rpm.
```

**Review by smlambert — APPROVED** [review]
LGTM, apart from a couple of minor acronym suggestions

**Review by karianna — APPROVED** [review]
- [x] LGTM - does the installer page on Adoptium.net also need updating?

**Comment by gdams** [comment]
@karianna I will follow up with that. I've added the docs to my new website remake but will add to the old website for now

**Pull request merged**